package.json

```diff
@@ -24,7 +24,7 @@
     "css": "npm-run-all css-compile css-prefix css-minify",
     "css-compile": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 scss/ -o dist/css/",
     "css-lint": "npm-run-all --continue-on-error --parallel css-lint-*",
-    "css-lint-stylelint": "stylelint \"**/*.{css,scss}\" --cache --cache-location .cache/.stylelintcache",
+    "css-lint-stylelint": "stylelint \"**/*.{css,scss}\" --cache --cache-location .cache/.stylelintcache--report-needless-disables",
```

Comment:
Isn't a space necessary before the `--report-needless-disables`?

I tried run `npm run lint` after fix this and remove my `.cache` folder, but stylelint reported a lot of errors:

![image](https://user-images.githubusercontent.com/4065765/65705523-9a17b400-e0c3-11e9-9a5e-2a4a2c78220a.png)
and more.

Reply:
refs: https://github.com/stylelint/stylelint/issues/4203

Reply:
Ooops, the space should be there.

Reply:
But yeah, I see now the issue, hence why i hadn't enabled it before :)

I'll drop it for now.